packages/client/test/rpc/web3/clientVersion.spec.ts

```diff
@@ -1,36 +1,34 @@
 import { platform } from 'os'
 import { assert, describe, it } from 'vitest'
 
-import { baseRequest, baseSetup, params } from '../helpers'
+import { baseSetup } from '../helpers.js'
 
 const method = 'web3_clientVersion'
 
 describe(method, () => {
   it('call', async () => {
-    const { server } = baseSetup()
+    const { rpc } = baseSetup()
 
-    const req = params(method, [])
-    const expectRes = (res: any) => {
-      const { result } = res.body
-      const { version } = require('../../../package.json')
-      const expectedClientTitle = 'EthereumJS'
-      const expectedPackageVersion = version
-      const expectedPlatform = platform()
-      const expectedNodeVersion = `node${process.version.substring(1)}`
+    const res = await rpc.request(method, [])
 
-      let msg = 'result string should not be empty'
-      assert.notEqual(result.length, 0, msg)
-      const [actualClientTitle, actualPackageVersion, actualPlatform, actualNodeVersion] =
-        result.split('/')
-      msg = 'client title should be correct'
-      assert.equal(actualClientTitle, expectedClientTitle, msg)
-      msg = 'package version should be correct'
-      assert.equal(actualPackageVersion, expectedPackageVersion, msg)
-      msg = 'platform should be correct'
-      assert.equal(actualPlatform, expectedPlatform, msg)
-      msg = 'Node.js version should be correct'
-      assert.equal(actualNodeVersion, expectedNodeVersion, msg)
-    }
-    await baseRequest(server, req, 200, expectRes)
+    const { result } = res
+    const { version } = require('../../../package.json')
+    const expectedClientTitle = 'EthereumJS'
+    const expectedPackageVersion = version
+    const expectedPlatform = platform()
+    const expectedNodeVersion = `node${process.version.substring(1)}`
+
+    let msg = 'result string should not be empty'
+    assert.notEqual(result.length, 0, msg)
+    const [actualClientTitle, actualPackageVersion, actualPlatform, actualNodeVersion] =
+      result.split('/')
+    msg = 'client title should be correct'
+    assert.equal(actualClientTitle, expectedClientTitle, msg)
+    msg = 'package version should be correct'
+    assert.equal(actualPackageVersion, expectedPackageVersion, msg)
+    msg = 'platform should be correct'
+    assert.equal(actualPlatform, expectedPlatform, msg)
+    msg = 'Node.js version should be correct'
+    assert.equal(actualNodeVersion, expectedNodeVersion, msg)
```

Comment:
Again I think the code would be cleaner if we didn't redeclare msg on every other line.

Reply:
Good point.  Will clean this up later today.  Thanks for the review!